app/lib/common/utilities/pdf_preview.dart

```diff
@@ -0,0 +1,73 @@
+import 'dart:io';
+
+import 'package:flutter/material.dart';
+import 'package:flutter/widgets.dart';
+import 'package:flutter_full_pdf_viewer/full_pdf_viewer_scaffold.dart';
+import 'package:path_provider/path_provider.dart';
+import 'package:pdf/pdf.dart';
+import 'package:pdf/widgets.dart' as pw;
+
+class PdfPreview extends StatefulWidget {
+  @override
```

Comment:
We'll be able to pass in the ATR object in the future like so:
```suggestion
  final TransportRecord record;
  
  @override
```
And then access the record in `initState()` to update itself.